tests/scf-response3/input.dat

```diff
@@ -0,0 +1,30 @@
+# UHF Dipole Polarizability Test
```

Comment:
Can we test that for a closed-shell molecule, the UHF and RHF polarizabilities agree?

Reply:
Done.